roombasim/pittras/task/velocity_task.py

```diff
@@ -0,0 +1,47 @@
+'''
+velocity_task.py
+'''
+
+import numpy as np
+
+import roombasim.config as cfg
+
+from roombasim.ai import Task, TaskState
+
+
+class VelocityTask(Task):
+    '''
+    A task to accelerate/decelerate the drone to a given velocity. The task
+    accepts a single 3d vector that defines the target velocty.
+    '''
+
+    def __init__(self, target):
+        '''
+        target - 3d (v_x, v_y, v_z) vector
+        '''
+        self.target_xy = np.array(target[:2])
+        self.target_z = target[2]
+
+        # PID controller contstants
+        self.k_xy = cfg.PITTRAS_PID_XY
+
+        self.i_xy = np.array([0, 0], dtype=np.float64)
+        self.i_z = 0
```

Comment:
Do we need this?

Reply:
Nice catch. I didn't realize that the z axis is simply velocity at first.